src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -41,15 +41,13 @@ internal static void Init()
             //   * have high disk cost
 
             // We shouldn't create too many tasks.
-
-            // This task takes awhile, so it gets it's own task
+#if !UNIX
+            // Amsi initialize can be a little slow
             Task.Run(() =>
             {
-                // Building the catalog is expensive, so force that to happen early on a background thread, and do so
-                // on a file we are very likely to read anyway.
-                var pshome = Utils.DefaultPowerShellAppBase;
-                var unused = SecuritySupport.IsProductBinary(Path.Combine(pshome, "Modules", "Microsoft.PowerShell.Utility", "Microsoft.PowerShell.Utility.psm1"));
+                AmsiUtils.WinScanContent(content: string.Empty, sourceMetadata: string.Empty, warmUp: true);
```

Comment:
`Init()` only initialize the AMSI, but not creating a session. I think it's better to create the session as well. Also, if we only call `Init()` here, there is a slight chance of race condition in case the pipeline threads gets to `WinScanContent` and start to run `init` in a lock while the `init` in a background thread hasn't finished yet.